v2/basculehttp/frontDoor.go

```diff
@@ -91,7 +91,7 @@ func (fd *frontDoor) handleMissingCredentials(response http.ResponseWriter, err
 
 func (fd *frontDoor) handleInvalidCredentials(response http.ResponseWriter, err *bascule.InvalidCredentialsError) {
 	response.Header().Set("Content-Type", "text/plain")
-	response.WriteHeader(http.StatusBadRequest)
+	response.WriteHeader(http.StatusUnauthorized)
```

Comment:
That's handling credentials that were misformatted or unparseable.  That's definitely a bad request, since authentication hasn't even been done.

Reply:
gotcha, I thought anything invalid (like misformatted and unparseable) regarding creds would be consider a 401.